nautobot/core/tests/test_models.py

```diff
@@ -15,6 +16,12 @@ class FakeBaseModel(BaseModel):
         def clean(self):
             raise ValidationError("validation error")
 
+    class JSONFieldModel(BaseModel):
+        data = models.JSONField(null=True)
+
+        class Meta:
+            required_db_features = {"supports_json_field"}
```

Comment:
We don't currently have any plan to support a db that *doesn't* support JSONField, but I guess this doesn't hurt. :-)